assemble/pom.xml

```diff
@@ -297,11 +297,6 @@
       <artifactId>log4j-api</artifactId>
       <optional>true</optional>
     </dependency>
-    <dependency>
-      <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-core</artifactId>
-      <optional>true</optional>
-    </dependency>
     <dependency>
```

Comment:
We still need log4j-core in the assembly for runtime. That's why this one didn't have the same comment as the others.

Reply:
@ctubbsii - I was literally typing the same thing as you replied to this :)

Reply:
Fixed in 42a4fa2